contracts/lbp/LBPManager.sol

```diff
@@ -233,28 +255,36 @@ contract LBPManager {
     }
 
     /**
-     * @dev                             can pause/unpause trading
-     * @param _swapEnabled              enable/disable swapping
+     * @dev                             Can pause/unpause trading.
+     * @param _swapEnabled              Enables/disables swapping.
      */
-    function setSwapEnabled(bool _swapEnabled) public onlyAdmin {
+    function setSwapEnabled(bool _swapEnabled) external onlyAdmin {
         lbp.setSwapEnabled(_swapEnabled);
     }
 
     /**
-     * @dev     get required amount of project tokens to cover for fees and the actual LBP
+     * @dev     Get required amount of project tokens to cover for fees and the actual LBP.
      */
-    function projectTokensRequired()
+    function projectTokensRequired(uint8 _fundingTokenIndex)
         public
         view
         returns (uint256 projectTokenAmounts)
     {
-        projectTokenAmounts = amounts[0] + feeAmountRequired();
+        projectTokenAmounts =
+            amounts[_fundingTokenIndex] +
+            feeAmountRequired(_fundingTokenIndex);
     }
 
     /**
-     * @dev     get required amount of project tokens to cover for fees
+     * @dev     Get required amount of project tokens to cover for fees.
      */
-    function feeAmountRequired() internal view returns (uint256 feeAmount) {
-        feeAmount = (amounts[0] * primeDaoFeePercentage) / HUNDRED_PERCENT;
+    function feeAmountRequired(uint8 _fundingTokenIndex)
```

Comment:
I would follow the convention and let internal function be preceded with an underscore (or does solhint complain then)?

```suggestion
    function _feeAmountRequired(uint8 _fundingTokenIndex)
```

Reply:
No I didn't think about the convention. Changed it